packages/cli/src/index.js

```diff
@@ -119,6 +120,8 @@ async function run() {
     }
   }
 
+  if(config.typescript) config.ext = "ts"
```

Comment:
it looks here, like we are forcing the ".ts" extension on typescript parsing which makes perfect sense, however some people prefer using ".tsx" instead. I would recommend something in the line of:

if(config.typescript && !config.ext) config.ext = "ts";

to make sure that the user still manually can override the extension.
(if this is already covered, then i'm sorry about the interruption)

Reply:
+1

Reply:
Thks for the help :)